builder/builder-source-git.c

```diff
@@ -212,6 +213,43 @@ builder_source_git_extract (BuilderSource  *source,
   return TRUE;
 }
 
+static gboolean
+builder_source_git_bundle (BuilderSource  *source,
+                           BuilderContext *context,
+                           GError        **error)
+{
+  BuilderSourceGit *self = BUILDER_SOURCE_GIT (source);
+
+  g_autofree char *location = NULL;
+  g_autoptr(GFile) mirror_dir = NULL;
+  g_autofree char *mirror_dir_path = NULL;
+  g_autofree char *app_dir_path = g_file_get_path (builder_context_get_app_dir (context));
```

Comment:
Nitpick: you can initialize `app_dir_path` right before `mirror_dir_path` instead